ginga/rv/plugins/MultiDim.py

```diff
@@ -91,7 +92,8 @@ def __init__(self, fv, fitsimage):
         # Load plugin preferences
         prefs = self.fv.get_preferences()
         self.settings = prefs.create_category('plugin_MultiDim')
-        self.settings.set_defaults(auto_start_naxis=False)
```

Comment:
@Cadair , just want to double check that removing this `auto_start_naxis` option does not break anything that SunPy is doing. It is not used here in Ginga. If you use it on your side, I would recommend to subclass this plugin instead and add the option over there.

Reply:
Speak now for forever hold your peace! :wink: